src/wallet/wallet.h

```diff
@@ -666,7 +673,8 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     CAmount GetDebit(const CTransaction& tx, const isminefilter& filter) const;
     void chainStateFlushed(const CBlockLocator& loc) override;
 
-    DBErrors LoadWallet();
+    enum class do_init_used_flag { Init, Skip };
```

Comment:
Does it make sense to keep the naming of this enum more generic to cover the initialization of the entire address book than just the used attribute?

e.g. `enum class AddressBook { Initialize, Skip_Initialization }`

Reply:
3ee478b9cba305dc87de1ad093ca14a89e58b516 it looks like this codebase uses PascalCase for enum classes, and doing so would also make the code more readable, for example easier to distinguish the PascalCase type from the snake_cased param name in function definitions

Reply:
It's a weird hack made necessary only by an oddity in the wallet tool. I'm not sure it makes sense to guess about what other hacks might or might not be needed in the future.